src/tequila/quantumchemistry/chemistry_tools.py

```diff
@@ -165,7 +165,7 @@ def fermionic_excitation(self, angle: typing.Union[Real, Variable, typing.Hashab
             if isinstance(control, int):
                 crt.append(control)
             else:
-                crt = crt + control
+                crt = crt + [*control]
```

Comment:
shouldn't make a difference (but also shouldn't break anything). 
Where did you have issues with the first line?